foodalert/sender.py

```diff
@@ -46,22 +47,25 @@ def send_amazon_sms(recipient, message, **kwargs):
     def format_message(message):
         event = message['event']
         foods = message['food']['served']
-        text = "A new Hungry Husky Event: '" + event + "' has been posted! \n"
-        time = datetime.strftime(message['time']['end'], "%c")
+        text = foods + " leftover from " + event + ".\n\n"
```

Comment:
Should change this to something like "Food leftover from ______" (event) because we don't really know what a host might be filling in for the foods on the form.